tests/queries/custom_aggregate/query.trickle

```diff
@@ -0,0 +1,22 @@
+define tumbling window by_two
+with
+  size = 2
+end;
+
+mod m with
+    aggregate collect_joined of
```

Comment:
reading this I am wondering if `define aggregate` would be better here,  since it's a troy level entity we use `define` for any other troy entity. OTOH, `fn` doesn't use define and while `fn` is in tremorsciprt level there is also a argument to keep them the same.